Runtime/KongregateWeb.cs

```diff
@@ -423,7 +428,7 @@ private static void AssertIsReady()
     {
         AssertInstanceExists();
 
-        if (!_instance._kongregateApiLoaded && IsKongregateAPIAvailable())
+        if (!_instance._kongregateApiLoaded && IsKongregateAPIAvailable)
```

Comment:
I'd suggest checking `_kongregateApiAvailable` directly here, since going through the `IsKongregateAPIAvailable` property will re-run the `AssertInstanceExists()` check.

Reply:
Oh wait, I can make suggestions directly on GitHub 🎉 

```suggestion
        if (!_instance._kongregateApiLoaded || !_instance._kongregateApiAvailable)
```

I'm also suggesting that you change it to throw the exception if the API is not available, since that's logically the same as the API not being ready.